assets/twitch-player.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <script src="https://player.twitch.tv/js/embed/v1.js"></script>
+    <title>Twitch Player</title>
+  </head>
+  <body>
```

Comment:
I think you can get rid of the `margin-left` and `margin-top` by removing the margin on the body and html tags: `style="margin: 0; padding: 0;"`

Reply:
Done!